### PR TITLE
#21 feat: add render table's cells with correct info from db

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/model/firebaseCollection/time/Time.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import {TimeUnit} from "src/model/businessModel/time/timeUnit/TimeUnit";
+
+/**
+ * What was done
```
Comment (tritonJS826) [inline mw-webapp/src/model/firebaseCollection/time/Time.ts:4]: What)

File: mw-webapp/src/model/firebaseCollection/time/Time.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+export class Time {
+
+  /**
+   * Unit of time measurement
+   */
+  public timeUnit: TimeUnit;
+
+  /**
+   * Number of time units (minutes) spent doing the job
```
Comment (tritonJS826) [inline mw-webapp/src/model/firebaseCollection/time/Time.ts:14]: Not minutes

**Review by tritonJS826 — APPROVED** [review]

**Review by SergioAJS — APPROVED** [review]

File: mw-webapp/src/model/firebaseCollection/time/Time.ts (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/model/firebaseCollection/time/Time.ts:4]: Change comment to "How long was the job done"

File: mw-webapp/src/model/firebaseCollection/time/Time.ts (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/model/firebaseCollection/time/Time.ts:14]: Deleted "minutes"

**Pull request merged**